styles/stylesheet.css

```diff
@@ -0,0 +1,434 @@
+  text-align: left;
+  grid-column: 2;
+  grid-row: 1 / 2;
+}
+
+.dog-3 {
+  border: 2px solid #d3d3d3;
+  text-align: left;
+  grid-column: 3;
+  grid-row: 1 / 2;
+}
+
+.dog-4 {
+  border: 2px solid #d3d3d3;
+  text-align: left;
+  grid-column: 4;
+  grid-row: 1 / 2;
+}
+
+.dog-5 {
+  border: 2px solid #d3d3d3;
+  text-align: left;
+  grid-column: 1 / 2;
+  grid-row: 3 / 4;
+}
+
+.dog-6 {
+  border: 2px solid #d3d3d3;
+  text-align: left;
+  grid-column: 2;
+  grid-row: 3 / 4;
+}
+
+.dog-7 {
+  border: 2px solid #d3d3d3;
+  text-align: left;
+  grid-column: 3;
+  grid-row: 3 / 4;
+}
+
+.dog-8 {
+  border: 2px solid #d3d3d3;
+  text-align: left;
+  grid-column: 4;
+  grid-row: 3 / 4;
+}
+
+.dog-9 {
+  border: 2px solid #d3d3d3;
+  text-align: left;
+  grid-column: 1 / 2;
+  grid-row: 5 / 6;
+}
+
+.dog-10 {
+  border: 2px solid #d3d3d3;
+  text-align: left;
+  grid-column: 2;
+  grid-row: 5 / 6;
+}
+
+.dog-11 {
+  border: 2px solid #d3d3d3;
+  text-align: left;
+  grid-column: 3;
+  grid-row: 5 / 6;
+}
+
+.dog-12 {
+  border: 2px solid #d3d3d3;
+  text-align: left;
+  grid-column: 4;
+  grid-row: 5 / 6;
+}
+
+.blog {
+  text-align: center;
+}
+
+.blog-content {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  /*margin: .5em;*/
+  justify-content: space-between;
+  position: relative;
+}
+
+.blog-img {
+  padding: .25em;
+  margin: 0;
+  /*margin: .5em;*/
+  float: left;
+}
+
+.blog-header {
+  font-family: 'Domine', serif;
+  font-size: 2em;
+  color: #083c60;
+}
+
+.blog-text {
+}
+
+.more {
+  font-family: 'Roboto', sans-serif;
+  font-size: 2em;
+  color: #083c60;
+  background-color: #d9d9d9;
+  padding: .25em;
+  margin: 0 /*auto*/;
+  text-align: center;
+  max-width: 33%;
+}
+
+footer {
+  background-color: #3d3d3d;
+  color: #ffffff;
+  font-size: .5em;
+  padding: 2em;
+  border: none;
+  margin: 0;
+  /*margin: 1em;*/
+  text-align: center;
+  width: 100%;
+}
+
+.footer {
+  color: #ffffff;
+  font-size: .75em;
+}
+
+.footer-logo {
+  display: block;
+  margin: 0;
+  /*margin: auto;*/
+  width: 25%;
+  }
+
+.footer-nav {
+  color: #ffffff;
+  padding: .5em;
+  margin: 0;
+  /*margin: .5em;*/
+  font-family: 'Roboto', sans-serif;
+  text-decoration: none;
+  text-align: center;
+}
+
+.name, .email, .address, .city, .state, .zip, .pick-up, .first, .adopted, .total {
+  font-family: 'Domine', serif;
+  font-weight: 900;
+  font-size: 1.5em;
+  color: #083c60;
+}
+
+.cost-2 {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 900;
+}
+
+button, input, select, textarea {
+  font-family: inherit;
+  font-size: 100%;
+  box-sizing: border-box;
+  display: block;
+  margin: 0;
+  /*margin: 1em;*/
+}
+
+form {
+  padding: 1rem;
+  margin: 0;
+  /*margin: 1rem;*/
+}
+
+fieldset {
+  border: none;
+}
+
+fieldset input {
+  display: inline;
+}
+
+input[type=submit] {
+  width: auto;
+  color: #ffffff;
+  background-color: #81b741;
+}
+
+.cart {
+  display: flex;
+  flex-flow: wrap row;
+  margin: 0;
+  /*margin: 1em;*/
+}
+
+button {
```

Comment:
I would modify this slightly to sit at the bottom and take up 100% of the width.